tools/wurf_options.py

```diff
@@ -63,6 +66,20 @@ def resolve(ctx):
 
 class WurfOptions(Options.OptionsContext):
 
+    def execute(self):
+
+        # Execute the context, this will go out and invoke the option(...)
+        # function defined in the wscript. We have some extra tools that
+        # also define the option(...) function to avoid having to call load
+        # in all wscripts to initialize those, we can do it here.
+
+        self.load('wurf_common_tools')
```

Comment:
To the first, if you don't want to load the tools, then probably you should use a different waf (e.g. the stock one). I think that would be fine. 

It is easy to add such that they will be loaded in both Options, Resolve, Configuration and Build contexts. That is probably a good idea (if needed I will check that).